### ISSUE TITLE
Reproduce Issue - Loss

### ISSUE THREAD

**Issue by YangshuaiWang** [dlanzo/CRANE#8]
Dear developer,

Thank you very much for providing the code and data resources. We have been studying your recent paper "Unified machine-learning framework for property prediction and time-evolution simulation of strained alloy microstructure"carefully and are currently attempting to reproduce some preliminary results, only for the purpose of benchmark.

Following the instructions, we used the dataset provided at https://archive.materialscloud.org/records/s31sf-y9t40, specifically the files from training_set_0-99.zip to training_set_1500-1599.zip as our training data. We did not modify the network architecture or parameters from the original code.

For preprocessing, we arranged the contents of each .npy file in row format and appended the corresponding material parameters at the end of each row. The combined data file (table_comb.txt) is attached for your reference.

However, during training, we noticed that the loss function does not decrease, and the overall training performance differs significantly from the results shown in your paper. We have attached the plot of the loss evolution for your reference.

Could you please advise whether there might be any issues in our data processing or training pipeline? We would greatly appreciate your guidance on how to resolve this issue.

Thank you very much for your time and kind help.

Best regards,

<img width="329" height="247" alt="Image" src="https://github.com/user-attachments/assets/12725425-928d-47d3-92a3-b819ded2c6c7" />

[table_comb.txt](https://github.com/user-attachments/files/23112736/table_comb.txt)

**Comment by dlanzo** [comment]
Dear YangshuaiWang,

Thank you very much for taking the time to try to reproduce our results and for giving us feedback.

The `table_comb.txt` file seems fine, so the issue is probably related to the parameters passed to the `train.py` script.

If you launched the train script without any options, e.g. by `python3 train.py`, the script uses default training parameters, which are **NOT** the ones used in the paper. Those values actually come from another work and are there as placeholders. In a future release, we will probably try to add some more meaningful values.

You can verify this by inspecting the `train_logs/your_id/args.txt` file, which contains all training hyperparameters.

Consider the following `.sh` file containing the full specifications of non-default parameters used in the paper (modify the values of variables according to your system, in particular specify the proper `--train_set` and `--valid_set` paths).

[launch_training.sh](https://github.com/user-attachments/files/23163169/launch_training.sh)

Please, let me know if this solves the issue. Do not hesitate to point out other problems: we are a small team and all feedback from other users is most welcome!

**Comment by YangshuaiWang** [comment]
Thank you, although the issue hasn’t been fully resolved, I really appreciate your effort. I’ll go ahead and close this issue for now.
